app/.env.example

```diff
@@ -1,3 +1,9 @@
 VITE_APP_TIPS_ADDRESS=
 VITE_APP_BACKEND_URL=
 VITE_APP_ETHERSCAN_URL=
```

Comment:
```suggestion
```

we can remove and use `VITE_APP_BLOCK_EXPLORER` instead, and dont forget to change the other code that use `VITE_APP_ETHERSCAN_URL`

Reply:
Okay, if you have code that depends on that variable you can update once changes have been approved so I don't have to dig into the code looking for where it's used.